allauth/templates/account/email/base_notification.txt

```diff
@@ -0,0 +1,24 @@
+{% extends "account/email/base_message.txt" %}
+{% load i18n %}
+{% load account %}
+{% user_display user as user_display %}
+
+{% block content %}
+Hello {{ user_display }},
+
+We wanted to let you know that there has been a recent change to your account information.
+
+Here are the details of the change:
+
+{% block notification_message %}
+{% endblock notification_message%}
+
+Change details
+Time: {{timestamp}}
+IP address : {{ip}}
+Browser agent : {{browser_agent}}
+
+
+If this change was not made by you, please contact us immediately at support of {{site_name}}.
+
+{% endblock %}
```

Comment:
You have put `endblock` on a line of its own with an empty line just before it. I suspect that this results in too many white space before the "Thank you for using our site" footer. Can you have a look at how the final mail is formatted?